cytotable/convert.py

```diff
@@ -423,6 +425,10 @@ def _prepend_column_name(
 
     targets = tuple(metadata) + tuple(compartments)
 
+    # if we have no targets or metadata to work from, return the table unchanged
+    if len(targets) == 0:
+        return table_path
```

Comment:
Is there a way to provide feedback to the user that this happened? Well, actually, maybe a better question is: Would this ever be triggered if a user added targets/metadata but cytotable failed to include them?

Reply:
To my knowledge there's no way this could be triggered when a user adds targets/metadata to CytoTable arguments. This would only ever happen where there are none. I'll add a logging warning message for this if-branch.